ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -243,3 +243,13 @@ def compress(condition, a, *, axis=None, out=None):
         )
     arr = arr[: condition_arr.shape[0]]
     return ivy.moveaxis(arr[condition_arr], 0, axis)
+
+
+@to_ivy_arrays_and_back
```

Comment:
We only need to convert input to `ivy.Array` and as the returned values are Booleans we will used `input_to_ivy_array` decorator
This should work
```python
@inputs_to_ivy_arrays
def iterable(y):
    return hasattr(y, "__iter__") and y.ndim > 0
```

Reply:
Hi, really appreciate your feedback. Will practice pushing cleaner code like yours